CGF-Custom-HA-1NIC-AS-ELB-ILB-STD/azuredeploy.json

```diff
@@ -123,6 +123,7 @@
   "variables": {
     "computeApiVersion": "2017-03-30",
     "networkApiVersion": "2017-08-01",
+    "bloburi" : "[if(equals(contains(resourceGroup().location,'usgov'),'True'),'.blob.core.usgovcloudapi.net','.blob.core.windows.net')]",
```

Comment:
The variable was only implemented against the first of the two VMs created. Such that if I were using usgov, the first VM would probably deploy and the second would likely fail.